sample/kotlin/src/main/kotlin/com/datadog/android/sample/compose/NavigationSample.kt

```diff
@@ -129,3 +149,12 @@ internal sealed class Screen(
 internal fun <T> oneOf(vararg items: T): T {
     return items[Random.nextInt(items.size)]
 }
+
+private class DatadogLifecycleObserver : LifecycleEventObserver {
+    @OptIn(ExperimentalRumApi::class)
+    override fun onStateChanged(source: LifecycleOwner, event: Lifecycle.Event) {
+        if (event == Lifecycle.Event.ON_RESUME) {
+            GlobalRumMonitor.get().addViewLoadingTime()
```

Comment:
won't we have multiple calls for the same view here when application goes info background and then back into foreground?

Reply:
when the app goes into background there will be a new view...we stop the view on the `onPause` the way I remember.

Reply:
no, here view tracking relies on navigation - I don't think navigation events are fired when app changes the state (but I may be wrong). While `addViewLoadingTime` call here relies not on the navigation, but on the lifecycle events.

Reply:
let me double check this, you might be right, I was assuming we rely on the view lifecycle here